module/testing.py

```diff
@@ -27,10 +28,29 @@ def lock_path(display):
     return '/tmp/.X%d-lock' % display
 
 
+def get_lock(display):
+    lockfile = lock_path(display)
+    try:
+        f = open(lockfile, "w")
```

Comment:
This will leak f, can we do this as `with open(lockfile, "w") as f`?

Reply:
I thought I had read somewhere that the file descriptor must not be closed, but I cannot find where. I found other examples closing it, so it should be fine, I must have read too fast.